### DISCUSSION TITLE
Sources for winlogbeat-oss

### DISCUSSION THREAD

**Discussion by fgreinacher** [chocolatey-community/chocolatey-packages#2199] [Q&A]
Can somebody help me get the sources for the recent versions of the https://community.chocolatey.org/packages/winlogbeat-oss package? The package links to https://github.com/mkopnsrc/chocolatey-packages, but the last commit there happened nearly a year ago.

I don't fully understand how the publishing works, glad about any kind of hint 🙇

**Comment by pauby** [comment]
If you just need the Chocolatey scripts and .nuspec file, download the package from the package page.

**Reply by fgreinacher** [reply]
That's a good hint, thanks!

I would have hoped for a little more reproducibility here, most other packages I found provide Git tags.

**Reply by fgreinacher** [reply]
Also I could not find a GHA workflow or something similar, is it common to push new versions manually?

**Reply by TheCakeIsNaOH** [reply]
> I would have hoped for a little more reproducibility here, most other packages I found provide Git tags.

That up to the package maintainer to provide, it is not something required on the Community Repository.

> Also I could not find a GHA workflow or something similar, is it common to push new versions manually?

Some maintainers exclusively build and push manually, and some others primarily have automation to build and push packages. Again, completely up to the maintainer.

**Reply by pauby** [reply]
One suggestion would be to look at the https://github.com/chocolatey-community/chocolatey-packages/ repository, as it contains around 400 automated packages.

**Reply by fgreinacher** [reply]
> One suggestion would be to look at the https://github.com/chocolatey-community/chocolatey-packages/ repository, as it contains around 400 automated packages.

Yes, that repository looks how I'd expect it, with tagged commits for every released package version. But the package we need is not there 😢

**Reply by pauby** [reply]
@fgreinacher I didn't suggest you look at it for tagged commits. I suggested you look at it because it runs an automated system for updating packages because of this comment:

>  is it common to push new versions manually?

You're getting carried away with GHA workflows and tagged commits. Neither of those are important when your focus is how to automate your packages.

**Reply by fgreinacher** [reply]
Thanks for clarifying, I have seen this before, it's looking great and commits like https://github.com/chocolatey-community/chocolatey-packages/commit/5eb007314de9dafb62768fe1a80331b8165ef149 would fulfill my wish of traceability :)

**Comment by fgreinacher** [comment]
@mkopnsrc Can you maybe help here?
